### PR TITLE
added support for windows

### PR DESCRIPTION
**Pull request by RohanImmanuel**

### CODE REVIEW

File: lib/BrowserStackContext.php [hunk 1]
```diff
@@ -24,7 +24,7 @@ public static function setup()
         $CONFIG = $GLOBALS['CONFIG'];
         $task_id = getenv('TASK_ID') ? getenv('TASK_ID') : 0;
 
-        $url = "https://" . $GLOBALS['BROWSERSTACK_USERNAME'] . ":" . $GLOBALS['BROWSERSTACK_ACCESS_KEY'] . "@" . $CONFIG['server'] ."/wd/hub";
+        $url = "http://" . $GLOBALS['BROWSERSTACK_USERNAME'] . ":" . $GLOBALS['BROWSERSTACK_ACCESS_KEY'] . "@" . $CONFIG['server'] ."/wd/hub";
```
Comment (Archish27) [inline lib/BrowserStackContext.php:27]: Why not `https`?

File: composer.json [hunk 2]
```diff
@@ -10,9 +10,12 @@
     },
     "scripts": {
         "test": "composer single && composer local && composer parallel",
-        "single": "./bin/behat --config=config/single.conf.yml",
-        "local": "./bin/behat --config=config/local.conf.yml",
-        "parallel": "CONFIG_FILE=config/parallel.conf.yml /usr/bin/env php lib/parallel.php"
+        "single": "\"./bin/behat\" --config=config/single.conf.yml",
```
Comment (Archish27) [inline composer.json:13]: Was it not working previously?

File: lib/BrowserStackContext.php (hunk 1 above)
Comment (RohanImmanuel) [reply lib/BrowserStackContext.php:27]: Https caused errors on windows systems

File: composer.json (hunk 2 above)
Comment (RohanImmanuel) [reply composer.json:13]: this previously worked on mac and linux but failed on windows

File: lib/parallel.php [hunk 3]
```diff
@@ -12,14 +12,20 @@
 $procs = array();
 
 foreach ($CONFIG['environments'] as $key => $value) {
-    $cmd = "TASK_ID=$key ./bin/behat --config=". getenv("CONFIG_FILE")." 2>&1\n";
+    if(stripos(PHP_OS, 'WIN') === 0) {
```
Comment (Archish27) [inline lib/parallel.php]: will PHP_OS return win32 / win64 if yes `preg_match('/win/i', PHP_OS)`?

Comment (RohanImmanuel) [reply lib/parallel.php]: possible values for PHP_OS on windows systems include 
- WIN32
- WINNT
- Windows
ref: https://github.com/php/php-src/blob/0d51ebd1a54af59d915c551a240b56bb3f0e26a6/main/main.c#L2000

**Review by Archish27 — APPROVED** [review]

**Comment by Archish27** [comment]
LGTM

**Review by yashLadha — APPROVED** [review]

**Pull request merged**